### PR TITLE
Display results

### PR DESCRIPTION
**Pull request by danieldekerlegand**
This PR adds result displays for quizzes. I opted against showing results on a separate gradebook page, since it would've been too much work. The results are instead displayed for each quiz on a quiz results page. Students can go to see their grades, and teachers can see the grade breakdown for the class.

### CODE REVIEW

**Comment by kmaus** [comment]
Whenever I viewed a quiz as an administrator, the students percentage was 100%, but it showed as an F. 
image

and once another student takes the course, the table doesn't create a new row
image

File: src/app/views/quizzes/score.html.erb [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        <th>Id</th>
+        <th>Name</th>
+        <th>Email</th>
+        <th>Score</th>
+        <th>Percentage</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <% @results.each do |result| %>
```
Comment (kmaus) [inline src/app/views/quizzes/score.html.erb]: I think you need to move this line before the <tr> tag to get the rows to behave properly.

File: src/app/views/quizzes/score.html.erb [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+        var gradeCount = {
+          a: 0,
+          b: 0,
+          c: 0,
+          d: 0,
+          f: 0
+        };
+
+        pieChartData.forEach(function(result) {
```
Comment (kmaus) [inline src/app/views/quizzes/score.html.erb]: Does result need to be `(result.marks.to_f / result.possible_marks) * 100.0`?

**Comment by danieldekerlegand** [comment]
I'm working on a fix for those two issues. I'll submit them and drop a line in Slack. I won't be available between 10 and 12 though, unfortunately, but I should have some time right before class.

**Comment by kmaus** [comment]
Sounds good, I was right with the moving result outside the <tr> tag, but my other idea didn't work.

**Review by kmaus — APPROVED** [review]

**Pull request merged**